### PR TITLE
twotorial: copy suggestions

### PR DESCRIPTION
**Pull request by dominic**

### CODE REVIEW

**Comment by jeffomatic** [comment]
👍 on the updates so far

File: dashboard/src/features/tutorial/steps.json [hunk 1]
```diff
@@ -5,163 +5,136 @@
...
     "route": "mockhsms/create",
     "image": "mockhsm",
     "content": [
-      {
-        "line": "Cryptographic private keys are the primary authorization mechanism on a blockchain."
-      },
-      {
-        "line": "They control both the issuance and transfer of asset units. An asset or account will define a single key required for issuance or transfers."
-      }
+      "Cryptographic private keys are the primary authorization mechanism on a blockchain. They control both the issuance and transfer of asset units. An asset or account will define a one or more keys required for issuance or transfers.",
```
Comment (chrisgarvin) [inline dashboard/src/features/tutorial/steps.json]: An asset or account will define ~a~ one or more keys

File: dashboard/src/features/tutorial/steps.json [hunk 2]
```diff
@@ -5,163 +5,136 @@
...
     "content": {
       "header": "Create MockHSM key",
       "steps": [
         {
-          "title": "Type a name for key alias (ex. 'gold', 'silver', 'bronze')",
+          "title": "Enter an alias for your key (ex. 'goldkey')",
           "description": [
             {
-              "line": "A key alias is a label used to distinguish between keys. We will use this key in the next step."
+              "line": "An alias is friendly label used to distinguish between keys. We will use this alias in the next step."
```
Comment (chrisgarvin) [inline dashboard/src/features/tutorial/steps.json]: An alias is **a** friendly label

File: dashboard/src/features/tutorial/steps.json [hunk 3]
```diff
@@ -5,163 +5,136 @@
...
     "image": "account",
     "content": [
-      {
-        "line": "An account is an object in Chain Core that tracks ownership of assets on a blockchain by creating and tracking control programs."
-      },
-      {
-        "line": "When you create an account, you provide one or more “root” keys and a quorum."
-      }
+      "An account is an object in Chain Core that tracks ownership of assets on a blockchain by creating and tracking control programs. The account object does not exist on the blockchain - it is local to this Chain Core. Only the control programs created in the account are visible on the blockchain.",
+      "We'll be moving assets from on account to another; let's create the first one now."
```
Comment (chrisgarvin) [inline dashboard/src/features/tutorial/steps.json]: We'll be moving assets from **one** account to another

**Comment by dominic** [comment]
thanks @chrisgarvin !

File: dashboard/src/features/tutorial/steps.json [hunk 4]
```diff
@@ -1,189 +1,145 @@
...
-    "button": null,
-    "dismiss": "End tutorial",
-    "route": null,
+    "title": "Tutorial - Creating Keys - Step 1 of 7",
     "content": {
       "header": "Create MockHSM key",
       "steps": [
         {
-          "title": "Type a name for key alias (ex. 'gold', 'silver', 'bronze')",
+          "title": "Enter an alias for your key (ex. 'goldkey')",
```
Comment (chrisgarvin) [inline dashboard/src/features/tutorial/steps.json]: thoughts on double-quote here for "goldkey"?

File: dashboard/src/features/tutorial/steps.json [hunk 5]
```diff
@@ -1,189 +1,145 @@
...
-            {
-              "line": "A key alias is a label used to distinguish between keys. We will use this key in the next step."
-            }
+            "An alias is a friendly label used to distinguish between keys. We will use this alias in the next step."
           ]
         },
         {
-          "title": "Click 'Submit' button",
-          "description": null
+          "title": "Click the 'Submit' button"
```
Comment (chrisgarvin) [inline dashboard/src/features/tutorial/steps.json]: Looks like I left single-quotes all throughout on these titles, my bad

**Pull request merged**